src/SFA.DAS.EmployerCommitmentsV2.Web/Mappers/Cohort/DetailsViewModelMapper.cs

```diff
@@ -111,7 +111,10 @@ private async Task<IReadOnlyCollection<DetailsViewCourseGroupingModel>> GroupCou
                             StartDate = a.StartDate,
                             OriginalStartDate = a.OriginalStartDate,
                             HasOverlappingEmail = emailOverlaps.Any(x=>x.Id == a.Id),
-                            ULN = a.Uln
+                            ULN = a.Uln,
+                            IsComplete = !(string.IsNullOrWhiteSpace(a.FirstName) || string.IsNullOrWhiteSpace(a.LastName) || a.DateOfBirth == null ||
+                                                                        string.IsNullOrWhiteSpace(a.CourseName) || a.StartDate == null || a.EndDate == null || a.Cost == null ||
```

Comment:
this could be a method to make it more readable "ApprenticeshipRequiredFieldsComplete" or something like that maybe?